Level2/NFT-Smart-Contract/Readme.md

```diff
@@ -7,9 +7,9 @@
 Funtions in the smart contract:-
```

Comment:
Do one last thing .

Make this line  7 as heading 
in line 8 make approve as  a bold and  do the same the rest of the lines like in line 9 make MintNfT  , safeTransferFrom etc as a bold

Reply:
@Vikash-8090-Yadav Done Sir